src/main/java/tasks/InverseFloatingArray.java

```diff
@@ -1,23 +1,23 @@
 package tasks;
 
-import utils.FileUtils;
-
 import java.io.IOException;
 import java.util.ArrayList;
 import java.util.List;
 
 public class InverseFloatingArray {
 
+    private TaskCore taskCore;
+
     private List<String> outputMessages = new ArrayList<>();
 
     private String className = InverseFloatingArray.class.getSimpleName();
 
     public void readInput() throws IOException {
-        List<String> input = FileUtils.readFromTxt(className);
+        List<String> input = taskCore.readFromTxt(className);
 
         input.forEach(this::inverseArray);
 
-        FileUtils.writeToTxt(className, outputMessages);
+        taskCore.writeToTxt(className, outputMessages);
     }
 
```

Comment:
Create method in interface:

boolean validateInput(String line);

create implementation in each class that will validate your input line accordingly!